telegram/ext/_basepersistence.py

```diff
@@ -398,30 +398,30 @@ def _insert_bot(self, obj: object, memo: Dict[int, object]) -> object:  # pylint
     def get_user_data(self) -> DefaultDict[int, UD]:
         """Will be called by :class:`telegram.ext.Dispatcher` upon creation with a
         persistence object. It should return the ``user_data`` if stored, or an empty
-        :obj:`defaultdict(telegram.ext.utils.types.UD)` with integer keys.
+        :obj:`defaultdict(telegram.ext._utils.types.UD)` with integer keys.
```

Comment:
see this is what I was talking about, user will not be able to know what type is `UD`!